mobile/src/people/departments/departments-horizontal-scrollable-list.tsx

```diff
@@ -1,35 +1,45 @@
 import React, { Component, SyntheticEvent } from 'react';
-import { Animated, Easing, View, Text, ScrollView, Dimensions, TouchableOpacity, NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
+import { Animated, Easing, View, Text, ScrollView, Dimensions, TouchableOpacity, NativeScrollEvent, NativeSyntheticEvent, ScrollViewStatic } from 'react-native';
 import { AppState } from '../../reducers/app.reducer';
 import { EmployeeCardWithAvatar } from '../employee-card-with-avatar';
-import { DepartmentsTreeNode, stubIdForSubordinates } from './departments-tree-node';
 import { Employee } from '../../reducers/organization/employee.model';
-import { PeopleActions, updateDepartmentIdsTree } from '../../reducers/people/people.action';
-import { loadEmployeesForDepartment } from '../../reducers/organization/organization.action';
 import { EmployeesStore } from '../../reducers/organization/employees.reducer';
+import { Department } from '../../reducers/organization/department.model';
 
 interface DepartmentsHScrollableListProps {
     treeLevel?: number;
-    departmentsTreeNodes?: DepartmentsTreeNode[];
-    headDepartment: DepartmentsTreeNode;
+    headDepartmentId?: string;
+    headDepartmentChiefId?: string;
+    departments?: Department[];
+    focusOnDepartmentWithId?: string;
     employees?: EmployeesStore;
     topOffset?: number;
     requestEmployeesForDepartment: (departmentId: string) => void;
-    updateDepartmentIdsTree: (index: number, department: DepartmentsTreeNode) => void;
+    updateDepartmentsBranch: (departmentId: string) => void;
     onItemClicked: (e: Employee) => void;
     employeesPredicate: (employee: Employee) => boolean;
 }
 
+interface AAScrollViewComponent extends Component {
```

Comment:
what AA stands for?

Reply:
safety naming in case RN team or 3rd libs name their stuff same way. AA - arcadia assistant.

Reply:
it's meaningless, JS supports modules (import/export), collisions should be solved differently. And it's even private. Please remove this prefix.

Reply:
addressed